crates/ika-types/src/messages_dwallet_mpc.rs

```diff
@@ -237,7 +240,7 @@ pub type AsyncProtocol = twopc_mpc::secp256k1::class_groups::AsyncProtocol;
 pub struct DWalletSessionEvent<E: DWalletSessionEventTrait> {
     pub epoch: u64,
     pub session_object_id: ObjectID,
-    pub session_type: SessionType,
+    pub session_type: SessionSource,
```

Comment:
if you rename the struct, rename the variable

Reply:
Why to be inconsistent with the move struct name?